Makefile

```diff
@@ -145,11 +147,13 @@ endif
 
 .PHONY: test
 test:
+	$(MAKE) -C ./tests/lenses build
```

Comment:
todo: this will change according to our discussion in having this under `make test:all` only right?

Reply:
Yes

Reply:
Sorry read the todo in readme

Reply:
> Sorry read the todo in readme

I dont understand - I should read a todo in the readme? Which todo? Why?

Reply:
> I dont understand - I should read a todo in the readme? Which todo? Why?

I meant that I just read the PR description, sorry

Reply:
Ah no worries :)